weave/panels/panel_plot.py

```diff
@@ -812,6 +812,14 @@ def predicate(row: graph.Node[types.TypedDict]):
             selection_max = selection[1]  # type: ignore
 
             if (
+                weave.types.TypedDict({"start": weave.types.Timestamp()}).assign_type(
```

Comment:
please add a comment to this condition block explaining why it is so specific. This is technically a hack and there might be a more general solution - we should flag that for future devs. For example: the correct thing might be to make the incoming `target` ungrouped. The implementation is fine, just please comment in detail why there is a start/stop etc...